keras_core/backend/numpy/math.py

```diff
@@ -298,3 +299,7 @@ def istft(
     else:
         end = expected_output_len
     return x[..., start:end]
+
+
+def rsqrt(x):
+    return np.array(jax_rsqrt(x))
```

Comment:
Why not `1. / sqrt(x)`? It's numpy native, and we're not worried about performance for the numpy backend.

Reply:
I was thinking of being consistent with other backends, but it should be okay to use `1. / sqrt(x)`
Fixed.